src/sentry/static/sentry/app/views/organizationGroupDetails/groupMerged/mergedList.tsx

```diff
@@ -1,27 +1,27 @@
 import React from 'react';
 import styled from '@emotion/styled';
-import PropTypes from 'prop-types';
 
 import EmptyStateWarning from 'app/components/emptyStateWarning';
 import Pagination from 'app/components/pagination';
 import {Panel} from 'app/components/panels';
 import QueryCount from 'app/components/queryCount';
 import {t} from 'app/locale';
-import SentryTypes from 'app/sentryTypes';
+import {Item, Project} from 'app/types';
 
 import MergedItem from './mergedItem';
 import MergedToolbar from './mergedToolbar';
 
-class MergedList extends React.Component {
-  static propTypes = {
-    onUnmerge: PropTypes.func.isRequired,
-    onToggleCollapse: PropTypes.func.isRequired,
-    items: PropTypes.arrayOf(SentryTypes.Event),
-    pageLinks: PropTypes.string,
-    orgId: PropTypes.string.isRequired,
-    project: SentryTypes.Project.isRequired,
-  };
+// XXX: How do I handle isRequired vs not?
```

Comment:
In `static propTypes` there's a bunch of types marked as `isRequired`, how do I port this concept to TS?

Reply:
It's required by default, in order to make a type "optional", you can use a `?` e.g.

```
type Props = {
  onUnmerge: Function;
  onToggleCollapse: Function;
  items?: Item[];
};
```